### ISSUE TITLE
During the inference phase ,there were copy mismatches in the CFG and errors in the noise sampling sapce.

### ISSUE THREAD

**Issue by Jerry-Auto** [DiffusionAD/Flow-Planner#19]
1. The dataset uses repeat_interleave,but flag and x are directly repeated,resulting in inconsisitent coping.for example,[1,2,3,4] to[1,1,2,2,3,3,4,4] and[1,2,3,4,1,2,3,4],mismatched.
2. the initial noise sampling space must be consistent with that used during training;that is ,sampling must be performed before segmentation,and then segmentation must occur, rather than sampling directly,in the segmented sapce.
3. x_init = torch.randn((B, self.action_num, self.planner_params['action_len'], self.planner_params['state_dim']), device=self.device)   
to        x_0 = torch.randn((B,1,self.planner_params['future_len'], self.planner_params['state_dim']), device=self.device)
        x_init = traj_chunking(x_0, self.planner_params['action_len'], self.planner_params['action_overlap'])
        x_init = torch.cat(x_init, dim=1)
